### ISSUE TITLE
RLS configuration warning

### ISSUE THREAD

**Issue by ofttryaj** [neoclide/coc-rls#43]
RLS configuration option `use_crate_backlist` is deprecated: use `crate_backlist` instead, How to resolve

**Comment by pictca** [comment]
open Coc config file by `:CocConfig`  in nvim,vim
edit to
```json
{
	"rust.crate_blacklist": {
		"default": true,
		"description": "Don't index crates on the crate blacklist.",
		"scope": "resource",
		"type": "boolean"
	}
}
```

**Comment by pictca** [comment]
closes #43

**Comment by ofttryaj** [comment]
Thank you very much
